### PR TITLE
tikv: update cdc config

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Update cdc config in TiKV.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1274,16 +1274,32 @@ raftdb 相关配置项。
...
++ 默认值：512MB
 
 ### `incremental-scan-speed-limit`
 
 + 增量扫描历史数据的速度上限。
 + 默认值：128MB，即 128MB 每秒。
 
+### `incremental-scan-threads`
+
++ 用于执行增量扫描历史数据的线程个数。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1294]:
```suggestion
+ 用于增量扫描历史数据的线程个数。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1274,16 +1274,32 @@ raftdb 相关配置项。
...
++ 默认值：4，即 4 个线程
+
+### `incremental-scan-concurrency`
+
++ 增量扫描历史数据任务的最大并发执行个数。
++ 默认值：6，即最多并发执行 6 个任务
++ 注意：`incremental-scan-concurrency` 需要大于等于 `incremental-scan-threads`，否则会使得启动报错。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1301]: 是 TiKV 启动还是 TiCDC 启动呢？

Comment (overvenus) [reply tikv-configuration-file.md:1301]:
```suggestion
+ 注意：`incremental-scan-concurrency` 需要大于等于 `incremental-scan-threads`，否则会使得 TiKV 启动报错。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1274,16 +1274,32 @@ raftdb 相关配置项。
...
+### `incremental-scan-threads`
+
++ 增量扫描历史数据任务的线程个数。
++ 默认值：4，即 4 个线程
+
+### `incremental-scan-concurrency`
+
++ 增量扫描历史数据任务的最大并发执行个数。
++ 默认值：6，即最多并发执行 6 个任务
++ 注意：`incremental-scan-concurrency` 需要大于等于 `incremental-scan-threads`，否则会使得 TiKV 启动报错。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1301]:
```suggestion
+ 注意：`incremental-scan-concurrency` 需要大于等于 `incremental-scan-threads`，否则 TiKV 启动会报错。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**